dcpy/extract/ingest.py

```diff
@@ -0,0 +1,33 @@
+from dcpy.connectors.edm import recipes
+from . import PARQUET_PATH, models
+
+
+# doesn't strictly need to be recipe config, but at least config-like object
+def archive_raw_dataset(dataset: str, version: str | None = None) -> recipes.Config:
+    """Given dataset and potentially a semantic version, ingest it from its source and archive it in its
+    raw format, with metadata, in edm-recipes/raw_data"""
```

Comment:
`edm-recipes/{dataset}/{version}/raw_data` or `edm-recipes/raw_data`?

Reply:
the latter is what we're going with for now